### DISCUSSION TITLE
running picom tells me another composite manager is already running

### DISCUSSION THREAD

**Discussion by unnamed1248** [yshui/picom#798] [Q&A] [answered]
when i run picom it tells me another composite manager is already running
`[ 03/12/2022 16:06:23.271 session_init FATAL ERROR ] Another composite manager is already running` 
neofetch is attached
please note i have only used linux for 1 week now so this might be a extremely stupid question
![Screenshot from 2022-03-12 16-11-27](https://user-images.githubusercontent.com/85708106/158039413-575f6db2-d2a3-4ccb-bc73-8102d3df8b2b.png)

**Comment by n-ovux** [comment, marked as answer]
This is because the Desktop Environment your using (Gnome) has a compositor built-in. Hope this helps ^_^
